app/src/main/java/org/connectbot/service/ConnectivityReceiver.java

```diff
@@ -79,22 +79,14 @@ public void onReceive(Context context, Intent intent) {
 			return;
 		}
 
-		boolean noConnectivity = intent.getBooleanExtra(ConnectivityManager.EXTRA_NO_CONNECTIVITY, false);
-		boolean isFailover = intent.getBooleanExtra(ConnectivityManager.EXTRA_IS_FAILOVER, false);
+		NetworkInfo info = (NetworkInfo) intent.getExtras()
+				.get(ConnectivityManager.EXTRA_NETWORK_INFO);
```

Comment:
Can you update this to use [`NetworkManager.getActiveNetworkInfo()`](https://developer.android.com/reference/android/net/ConnectivityManager.html#getActiveNetworkInfo()) since this API is deprecated?